src/Apps/Artwork/Components/UnlistedBannerModal.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button, ModalDialog, Spacer, Text } from "@artsy/palette"
+import * as React from "react"
+
+interface UnlistedBannerModalProps {
+  show: boolean
+  onClose(): void
+}
+
+export const UnlistedBannerModal: React.FC<UnlistedBannerModalProps> = ({
+  show,
+  onClose,
+}) => {
+  if (!show) return null
+
+  return (
+    <ModalDialog
+      onClose={onClose}
+      title="Private Listings"
+      footer={
+        <Button onClick={onClose} width="100%">
+          OK
+        </Button>
+      }
+    >
+      <Text variant="sm" data-testid="unlisted-modal-text">
```

Comment:
It was super tricky to get the text without these test ids. But I saw that we were using these data-testids to facilitate that in a couple other places.

Reply:
It was not rendering or what was the problem? I wonder what that property is used for 🤔

Reply:
+++ yes definitely pro to the testID, I was running into the issue on my side too. 
It was because of the nested child elements which contained different parts of the text, needed a better matcher to find the full sentence and not just the text of the deepest element